python/python/tests/test_query.py

```
@@ -1065,20 +1067,24 @@ async def test_query_serialization_async(table_async: AsyncTable):
-        full_text_query=FullTextSearchQuery(columns=[], query="foo"),
-        with_row_id=False,
+    # TODO: FTS query is not a simple string now,
+    # it can be nested, we need to figure out how to represent it with pydantic
+    hack_query = MatchQuery("foo", "text")
+    hack_query = FullTextSearchQuery(
+        columns=None, query=FullTextQuery.from_inner(hack_query._inner)
     )
+    q = (await table_async.search("foo")).limit(10).to_query_object()
+    assert q.limit == 10
+    assert q.with_row_id is False
+    assert q.full_text_query.query._inner.query_type() == "match"
```

Pylint (3.3.7)

[warning] Access to a protected member _inner of a client class

(W0212)